namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-frontend-staging/resources/s3.tf

```diff
@@ -0,0 +1,40 @@
+module "cla_frontend_static_files_bucket" {
+  source                        = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.6"
```

Comment:
Hey! Are you okay to drop this version down to 4.5 for now. We're in the middle of upgrading our Terraform stack. I'll update it to 4.6 on Monday next week.

Reply:
Done